seqr/views/apis/project_api.py

```diff
@@ -329,6 +330,55 @@ def project_mme_submisssions(request, project_guid):
     })
 
 
+@login_and_policies_required
+def project_notifications(request, project_guid, read_status):
+    project = get_project_and_check_permissions(project_guid, request.user)
+    is_subscriber = project.subscribers.user_set.filter(id=request.user.id).exists()
+    if not is_subscriber:
+        max_loaded = _project_notifications(
+            project, request.user.notifications).aggregate(max_loaded=Max('timestamp'))['max_loaded']
```

Comment:
can't you just do 
`max_loaded = _project_notifications(project, request.user.notifications).aggregate(Max('timestamp'))`